src/node_external_reference.h

```diff
@@ -17,6 +17,11 @@ using CFunctionCallbackWithMultipleValueAndOptions =
              v8::Local<v8::Value>,
              v8::Local<v8::Value>,
              v8::FastApiCallbackOptions&);
+using CFunctionVoid = void (*)(v8::Local<v8::Value>);
+using CFunctionVoid2 =
```

Comment:
Not super excited about the name here but then again none of the macro names here are great.